[CODE] Assignment3/CLASS/DrawingObject.js

```diff
@@ -202,14 +202,8 @@ function DrawingSetup() {
 			j = j + 25
 		}
 	}
-    
-    /*for (var i = 0; i < 20; i++) {
-        var x = Math.random();
-        var y = Math.random();
-        var object = DrawingObject.Instance(Tree, vec2(x * 1000, 500 + y * 300), vec2(0.15, 0.15));
-        object.z = 5;
-        object.outline = true;
-    }*/
+	
+	// Delete Tree loop
     
```

Comment:
Good.
But, Because Tree is no longer used, I think that it is good to remove class file(Tree.js)

Reply:
Ok!

Reply:
I don't think so..

Reply:
> I don't think so..

Umm.....
I thought again, the Tree file seems to be used as one of examples.
So I think remaining this file is good.